src/coord/src/catalog/builtin_table_updates.rs

```diff
@@ -646,10 +644,16 @@ impl CatalogState {
             (occurred_at / 1_000_000_000).try_into().expect("must fit"),
```

Comment:
This is wrong now and needs fixing.

Reply:
I think either of the following would work:

```
        let dt = NaiveDateTime::from_timestamp(
            (occurred_at / 1_000).try_into().expect("must fit"),
            ((occurred_at % 1_000) * 1_000_000).try_into().expect("must fit"),
        );
```

```
        let dur = Duration::from_millis(occurred_at);
        let dt = NaiveDateTime::from_timestamp(
            (dur.as_secs()).try_into().expect("must fit"),
            (dur.subsec_nanos()).try_into().expect("must fit"),
        );
```